### DISCUSSION TITLE
Column filters in modal

### DISCUSSION THREAD

**Discussion by nemanjam0** [KevinVandy/material-react-table#517] [Ideas]
Is it possible to add at least export for MRT_TableHeadCellFilterContainer so it becomes possible to display filter components to some other place instead of below column header. For example to allow creation of custom modal for table filters

**Comment by KevinVandy** [comment]
Sure

**Reply by alinsalaru** [reply]
Hi @KevinVandy 

Thank you for exporting `MRT_TableHeadCellFilterContainer`, however when trying to use it in a new modal component, `showColumnFilters` must also be[ set to **true** .](https://github.com/KevinVandy/material-react-table/blob/c74f8147c4d6988aeeac0d6be471c15539894ea3/packages/material-react-table/src/head/MRT_TableHeadCellFilterContainer.tsx#L25).  
```
 initialState={{
          showColumnFilters: true,
        }}
```

This will render column filters in 2 UI sections(header columns & modal) and will not allow column filters to render if `showColumnFilters` is not _true_.


With [v2](https://github.com/KevinVandy/material-react-table/blob/v2/packages/material-react-table/src/inputs/index.ts) we can use any Filtering Fields with our own discretion it seems. Do you have any hints when that will be released?

**Reply by KevinVandy** [reply]
I wish I was releasing it already. Hopefully I'll publish an alpha this weekend

**Reply by alinsalaru** [reply]
Take all the time you need @KevinVandy. Kudos for all the awesome work you've done so far !

Found a workaround for the above, thanks to the customisable way MRT is build
```
<MaterialReactTable
  muiTableHeadCellProps={{ 
            sx: {
              '.MuiCollapse-root' : {
                display: 'none'
              }
            }
          }}
</MaterialReactTable>
```

Writing this out in case it also helps @nemanjam0
